src/components/DynamicFormik/DynamicFormik.stories.js

```diff
@@ -47,6 +47,17 @@ const inputs = [
     minValueLength: '2',
     maxValueLength: '20',
   },
+  {
+    key: 'date',
+    label: 'Date',
+    value: '2020-01-01',
+    type: 'date',
+    placeholder: 'yyyy-mm-dd',
+    helperText: 'date',
+    pattern: 'Y-m-d',
+    min: '2020-01-01',
```

Comment:
`min` and `max` aren't values that are currently coming from our DataDrivenComponent model in the service. I think we need to get the service team to update to include these fields. Min and max could also work for other inputs, like the number input that are currently using `minValueLength` so we should look into that if you haven't.